README.md

```diff
@@ -24,7 +24,7 @@ The following example builds a [Fedora ELN](https://docs.fedoraproject.org/en-US
 the command on.
 
 ```
-mkdir output
+mkdir -p output
```

Comment:
Don't hesitate to make this a separate commit as well. Or just group all minor changes to the README under one commit.